Partner Manager 2.16.0 (#200)
mulesoft/docs-partner-manager: modules/ROOT/pages/endpoint-https-send.adoc

`+https://www.mulesoft.com/sendRequest?name=something+`
|x

|Response Timeout (in milliseconds)

> |Maximum time, in milliseconds, for which the request element blocks the execution of the flow that is waiting for the HTTP response. The defauilt is 15000 milleseconds.

Reviewer: Do you need "in milliseconds" in the description if it's in the parameter?

> |Maximum time, in milliseconds, for which the request element blocks the execution of the flow that is waiting for the HTTP response. The defauilt is 15000 milleseconds.

Reviewer: ```suggestion
|Maximum time for the request element to block the execution of a flow that is waiting for the HTTP response. The default is 15000 milliseconds.
```

|x

|Retry interval (in seconds)

> |Frequency of reconnection attempts when unable to connect to the remote HTTP or HTTPS service. The default is 60 seconds.

Reviewer: Do you need to specify what is unable to connect?

|x

|Maximum retry attempts

> |Maximum number of reconnection attempts to make. If you don’t want Partner Manager to retry after an initial connection failure, set this value to `0`. The default is `3`.

Reviewer: ```suggestion
|Maximum number of reconnection attempts. To prevent Partner Manager from retrying after an initial connection failure, set this value to `0`. The default is `3`.
```
